package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ferdium-recipes",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
This is the wrong package.json to bump. The one you want is https://github.com/ferdium/ferdium-recipes/blob/main/recipes/messenger/package.json#L4

Reply:
Whoops! Let me fix.